doc/Learn/Markup/Binding-To-DataTemplates.md

```diff
@@ -0,0 +1,89 @@
+---
+uid: Uno.Extensions.Markup.BindToDataTemplate
+---
+
+# Binding to DataTemplates
+
+When working with `DataTemplate`s, accessing data or commands from the parent context can be challenging. `DataTemplate`s operate within their own scope, making it difficult to bind properties or trigger actions in the parent `DataContext`. This separation can lead to confusion, especially when dealing with nested templates and multiple levels of data context.
+
+In the following example, the parent `DataContext` is the `ViewModel`. The `ViewModel` contains an `ICommand RemoveItemCommand`. The code demonstrates how to access that `ICommand` inside the `ItemTemplate` of the `ListView`.
+
+```csharp
+this.DataContext(new ViewModel(), (page, vm)
+    => page
+    .Content(
+        new ListView()
+            .ItemsSource(() => vm.Items)
+            .ItemTemplate<Item>(item =>
+                new StackPanel()
+                    .Children(
+                        new TextBlock()
+                            .Text(() => item.Text),
+                        new Button()
+                            .Content("Delete")
+                            .CommandParameter(() => item)
+
+                            // Since we have access to the `page` and `vm` alias from the DataContext method
+                            // We can take advantage of them and use them on our binding expression
+                            .Command(x => x.Source(page)
+                                           .DataContext()
+                                           .Binding(() => vm.RemoveItemCommand)
+                            )
+                    )
+            )
+    )
+)
```

Comment:
could use less indentation:
```suggestion
this.DataContext(new ViewModel(), (page, vm) => page.Content(
    new ListView()
        .ItemsSource(() => vm.Items)
        .ItemTemplate<Item>(item =>
            new StackPanel()
                .Children(
                    new TextBlock().Text(() => item.Text),
                    new Button()
                        .Content("Delete")
                        .CommandParameter(() => item)

                        // Since we have access to the `page` and `vm` alias from the DataContext method
                        // We can take advantage of them and use them on our binding expression
                        .Command(x => x
                            .Source(page)
                            .DataContext()
                            .Binding(() => vm.RemoveItemCommand)
                        )
                )
        )
))
```